docs/vmprof.rst

```diff
@@ -77,7 +77,7 @@ After ``-m vmprof`` you can specify some options:
 * ``--web-auth`` - auth token for user name support in the server.
 
 * ``-p period`` - float that gives you how often the profiling happens
-  (the max is about 300 Hz, rather don't touch it).
+  (the max is about 300 Hz, rather don't touch it). This is in microseconds, the default is 0.00099
```

Comment:
It's actually in seconds. The meaning of the default value is 0.99 ms.

Reply:
Fixed.

Yea, this reinforces that it needs to be in the docs, because from reading the code I obviously badly misread it :P